apps/judicial-system/backend/src/app/modules/notification/subpoenaNotification.service.ts

```diff
@@ -0,0 +1,196 @@
+import { MessageDescriptor } from 'react-intl'
+
+import {
+  Inject,
+  Injectable,
+  InternalServerErrorException,
+} from '@nestjs/common'
+import { InjectModel } from '@nestjs/sequelize'
+
+import { IntlService } from '@island.is/cms-translations'
+import { EmailService } from '@island.is/email-service'
+import { type Logger, LOGGER_PROVIDER } from '@island.is/logging'
+import { type ConfigType } from '@island.is/nest/config'
+
+import { INDICTMENTS_COURT_OVERVIEW_ROUTE } from '@island.is/judicial-system/consts'
+import { NotificationType } from '@island.is/judicial-system/types'
+
+import { EventService } from '../event'
+import { Subpoena } from '../subpoena'
+import { DeliverResponse } from './models/deliver.response'
+import { Notification, Recipient } from './models/notification.model'
+import { BaseNotificationService } from './baseNotification.service'
+import { notificationModuleConfig } from './notification.config'
+import { strings } from './subpoenaNotification.strings'
+
+type subpoenaNotificationType =
```

Comment:
Good idea. Please consider moving this declaration to the file containing `notification.ts` in the type library and using the type in the dto, controller and service instead of `NotificationType`. That way you avoid the cast from `NotificationType` to `subpoenaNotificationType`. Also, that allows you to use the type in the `SubpoenaService` where the notification messages are sent to the queue.

Please also consider renaming the type to `SubpoenaNotificationType` as that is the naming convention used for types.